test/api-test.js

```diff
@@ -137,4 +139,18 @@ describe('RippleAPI', function() {
       _.partial(checkResult, getOrdersResponse, done));
   });
 
+  it('getOrderBook', function(done) {
+    const orderbook = {
+      base: {
```

Comment:
It looks like these amounts eventually get passed in verbatim, to the book_offers request